spec/models/manageiq/providers/vmware/infra_manager/refresher_spec.rb

```diff
@@ -469,6 +469,13 @@ def assert_specific_host
       :location           => "4dce5b88-623e8e7e-0dc0-00188b404015"
     )
 
+    expect(@host.host_storages.where(:accessible => false).count).to eq(1)
+    inaccessible_host_storage = @host.host_storages.where(:accessible => false).first
```

Comment:
Minor, but you can use `find_by(:accessible => false)` and drop the `.first` on this call.

Reply:
Done